src/Blueprint.php

```diff
@@ -172,4 +170,31 @@ public static function assertEquals(mixed $expected, mixed $actual, string $mess
     {
         Assert::assertEquals($expected, $actual, $message);
     }
+
+    private function getUsagePathAndLines(Layer $layer, string $objectName, string $target): null|ViolationReference
+    {
+        $dependOnObjects = array_filter(
+            $layer->getIterator()->getArrayCopy(), //@phpstan-ignore-line
+            fn (ObjectDescription $objectDescription): bool => $objectDescription->name === $objectName
+        );
+
+        /** @var ObjectDescription $dependOnObject */
+        $dependOnObject = array_pop($dependOnObjects);
+
+        $names = ServiceContainer::$nodeFinder->findInstanceOf(
+            $dependOnObject->stmts,
+            Name::class,
+        );
+
+        /** @var array<int, Name> $names */
+        /** @phpstan-ignore-next-line */
+        $names = array_values(array_filter($names, static fn (Name $name): bool => $name->toString() === $target));
```

Comment:
here I have a strange phpstan error, can't manage to remove this ignore, any clue?